content/api/relay-webhooks.apib

```diff
@@ -182,6 +180,17 @@ Once registered, your relay webhook HTTP endpoint will receive inbound SMPP mess
 
 Create a relay webhook by providing a **relay webhooks object** as the POST request body.
 
++ Data Structure
+    + name: (string) - User-friendly name
+    + target: (string, required) - URL of the target to which to POST relay batches
+    + auth_token (string) - Authentication token to present in the `X-MessageSystems-Webhook-Token` header of POST requests to target.
+    + match (object, required) - Restrict which inbound messages will be relayed to the target.
+        + protocol (string, required) - Inbound messaging protocol associated with this webhook.
```

Comment:
not required

Reply:
I marked these are required because there is a sample for creating an SMTP webhook and SMPP webhook. Each has different items as marked as required.

Reply:
it *is* required if you are making specifically an smpt one